libopenage/util/fixed_point_test.cpp

```diff
@@ -203,6 +205,74 @@ void fixed_point() {
 		TESTNOEXCEPT((FixedPoint<int64_t, 32>::from_float(3.25).sqrt()));
 		TESTNOEXCEPT((FixedPoint<uint64_t, 32>::from_float(-3.25).sqrt()));
 	}
+
+	// Pure FixedPoint trig tests
+	{
+		using TrigType = FixedPoint<int64_t, 32, int64_t>;
+
+		// Testing sin() and cos()
+		for (int i = -100'000; i <= 100'000; i++) {
```

Comment:
the loop runs for a very long time. Can we go with lower precision for speeding these up a bit?